pkg/storage/minio/saver.go

```diff
@@ -17,7 +17,7 @@ func (s *storageImpl) Save(ctx context.Context, module, vsn string, mod []byte,
 	dir := s.versionLocation(module, vsn)
 	modFileName := dir + "/" + "go.mod"
 	zipFileName := dir + "/" + "source.zip"
-	infoFileName := dir + "/" + vsn + ".info"
+	infoFileName := dir + ".info"
```

Comment:
This is incorrect because it's missing a `/` which will end up not nesting the info bytes inside the `dir` -- it should at least be like the above such as `dir + "/" + "source.info"` -- Which brings up the question, what's wrong with `dir + "/" + vsn + ".info"`? Besides consistency that is.

Reply:
This is especially incorrect, because in the near future `storage.Checker` will need to check the length of an object to equal 3